tests/test_lib_eventhandler_logging.py

```diff
@@ -0,0 +1,161 @@
+# -*- coding: utf-8 -*-
+
+"""
+This file tests:
+
+lib/eventhandler/logging.py
+"""
+from mock import mock
+from datetime import datetime
+from werkzeug.test import EnvironBuilder
+from flask import Request
+from testfixtures import log_capture
```

Comment:
The testfixtures seems to be missing.

Reply:
Yes, they are already in master from the YAML PR :-)

Reply:
Hm, so the build in the master will succeed.